src/algorithms/tracking/ActsToTracks.h

```diff
@@ -17,13 +19,36 @@ namespace eicrecon {
 
 using ActsToTracksAlgorithm =
     algorithms::Algorithm<
-      algorithms::Input<edm4eic::Measurement2DCollection, std::vector<ActsExamples::Trajectories>>,
-      algorithms::Output<edm4eic::TrajectoryCollection, edm4eic::TrackParametersCollection, edm4eic::TrackCollection>
+      algorithms::Input<
+        edm4eic::Measurement2DCollection,
+        std::vector<ActsExamples::Trajectories>,
+        std::optional<edm4eic::MCRecoTrackerHitAssociationCollection>
```

Comment:
Why do you use std::optional here, since you always pass in the "CentralTrackingRawHitAssociations" and the association between the raw and sim hits should exist for each raw hit?

Reply:
The associations only exist for simulation input. We want algorithms to work for non-simulation input too.

Reply:
Got it. Thanks.